perllib/FixMyStreet/DB/Result/User.pm

```diff
@@ -54,16 +62,10 @@ __PACKAGE__->add_columns(
     is_nullable   => 0,
     original      => { default_value => \"now()" },
   },
-  "title",
-  { data_type => "text", is_nullable => 1 },
-  "twitter_id",
-  { data_type => "bigint", is_nullable => 1 },
-  "facebook_id",
-  { data_type => "bigint", is_nullable => 1 },
-  "extra",
-  { data_type => "text", is_nullable => 1 },
   "area_ids",
   { data_type => "integer[]", is_nullable => 1 },
+  "oidc_ids",
+  { data_type => "text[]", is_nullable => 1 },
```

Comment:
Not that it matters much, but if you generate this on a fresh db schema created from scratch, it wouldn't move the other rows around.

Reply:
Good to know for next time 👍